Modules/Forum/classes/class.ilForumExporter.php

```diff
@@ -46,7 +46,23 @@ public function getXmlRepresentation($a_entity, $a_schema_version, $a_id)
 		return $xml;
 	}
 
-	/**
+    public function getXmlExportTailDependencies($a_entity, $a_target_release, $a_ids)
```

Comment:
LGTM in general, but looks like a different code style/indentation compared to the other lines in this file.

Reply:
Yes, it uses spaces. How would you like to have the PR being delivered?

Reply:
Spaces are okay. Feel free to apply PSR-2 to the whole file.

Option 2) Maybe you could contact me (e.g. via short email) if this has been merged, so I can fix the code formatting afterwards.

Reply:
Spaces are okay. Feel free to apply PSR-2 to the whole file.

Option 2) Maybe you could contact me (e.g. via short email) if this has been merged, so I can fix the code formatting afterwards.

Reply:
Actually I stopped to follow this "move the whole file" procedure currently completely. These changes cannot be merged up and down automatically and the whole procedure produces too much manual work.